packages/ag-charts-enterprise/src/error-bar/errorBar.ts

```diff
@@ -123,11 +142,29 @@ export class ErrorBars extends _ModuleSupport.BaseModuleInstance implements _Mod
         this.selection.each((node, datum, i) => this.updateNode(node, datum, i));
     }
 
+    private inheritProperties(
```

Comment:
Let's sync up, we normally handle these scenarios using theme configuration (error bars should work the same way).

Reply:
https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-enterprise/src/box-plot/boxPlotSeries.ts#L479-L485